vite.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'vite';
```

Comment:
We already have a `vite.config.js` in the repo. Please integrate this into that file to have everything in the same place.